apps/web/composables/useCarousel/useCarousel.ts

```diff
@@ -0,0 +1,61 @@
+import { onMounted, onBeforeUnmount } from 'vue';
+
+const handleArrowsVisibility = () => {
+  const nextButton = document.querySelector('.swiper-button-next') as HTMLElement;
+  const previousButton = document.querySelector('.swiper-button-prev') as HTMLElement;
+
+  if (window.innerWidth < 768) {
+    if (nextButton) nextButton.style.display = 'none';
+    if (previousButton) previousButton.style.display = 'none';
+  } else {
+    if (nextButton) nextButton.style.display = 'block';
+    if (previousButton) previousButton.style.display = 'block';
+  }
+};
+
+const applySwiperStyles = () => {
+  const nextButton = document.querySelector('.swiper-button-next');
+  const previousButton = document.querySelector('.swiper-button-prev');
+  const activePagination = document.querySelector('.swiper-pagination-bullet-active');
```

Comment:
**question**: Would this work if there were multiple hero sliders or would it only color the buttons of the first one?

Reply:
Should work with multiple ones on the index.vue component. The module comes included with the above classes. Not sure if this would work on other pages since I call this onMounted() inside index.vue.